go/signedexchange/signedexchange_test.go

```diff
@@ -222,3 +225,60 @@ func TestSignedExchangeBannedCertUrlScheme(t *testing.T) {
 		t.Errorf("non-{https,data} cert-url unexpectedly allowed in an exchange")
 	}
 }
+
+func TestVerify(t *testing.T) {
+	u, _ := url.Parse("https://example.com/")
+	header := http.Header{}
+	header.Add("Content-Type", "text/html; charset=utf-8")
+
+	const ver = version.Version1b2
+	e, err := NewExchange(ver, u, nil, 200, header, []byte(payload))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := e.MiEncodePayload(16); err != nil {
+		t.Fatal(err)
+	}
+
+	now := time.Date(2018, 1, 31, 17, 13, 20, 0, time.UTC)
```

Comment:
Would you add a comment what this is?

Reply:
Oh, it was just copied from an existing test case, but it doesn't have to be fixed timestamp.
Changed to `time.Now()`.

Reply:
Would a constant time be better for testing in general?

Reply:
Yeah maybe. (I was Chromium build sheriff on Mon and Tue, and saw tests failing because of the end of DST...)

Changed to a (better-named) fixed timestamp.